docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -226,10 +215,10 @@ EOF
 
 .Verification
 
-Compare the following outputs with the topology specific configuration.
+Compare the following outputs with the topology-specific service configuration.
```

Comment:
In the “Verification” section, it says “Compare the following outputs with the topology-specific service configuration.” “Does “the following outputs” refer to the outputs from the commands in the following steps, i.e., checking that the database imported correctly, etc.?

Reply:
cc @bogdando (I'm not sure how this was meant.)

Reply:
yes

Reply:
I was going to ask the same question :) Possibly rephrase  to something like "Compare the command outputs from the following steps with the topology-specific service configuration"?